build.sbt

```diff
@@ -585,43 +588,55 @@ lazy val allModules: Seq[ProjectReference] = Seq(
   http,
   kafka,
   `marshallers-jodatime`,
+  `legacy-avro-decimal-compat-model`,
+  `legacy-avro-decimal-compat-protocol`,
+  `legacy-avro-decimal-compat-encoders`,
+  `health-check-unary`,
+  `example-health-client`,
+  `example-health-server-monix`,
+  `example-health-server-fs2`,
```

Comment:
How do we decide which modules are "core" and which are not? Example projects don't seem very core to me.

Is `coreModules` actually the list of projects that the docs depend on? If so, let's rename it.

Reply:
I've renamed the `others` by something more explicit. Basically, the modules that are not part of the `coreModules` are those that cannot be cross-compiled to Scala 2.13.x.